src/commands/baseTypeConverters.ts

```diff
@@ -35,7 +35,7 @@ export const baseTypeConverters = {
       throw new TypeConversionError(`\`${disableCodeBlocks(value)}\` is not a valid user`);
     }
 
-    const user = client.users.cache.get(userId);
+    const user = client.users.cache.get(userId as Snowflake);
```

Comment:
Make the `getUserId` function return a `Snowflake` so you don't have to do this everywhere.

Reply:
A lot of the casts i did here were done to quickly fix compiler errors at a time where i did not intend to make this a PR at all. I will do this in the future once i fixed some other issues Zeppelin exposes

Reply:
Fixed